src/components/Accordion.js

```diff
@@ -36,7 +36,7 @@ const Accordion = React.createClass({
   renderContainer: function(item, i) {
     const { opened, created } = this.state;
     const containerClassName =
-          item.header.toLowerCase().replace(/\s/g, "-") + "-pane";
+      `${item.header.toLowerCase().replace(/\s/g, "-")}-pane`;
```

Comment:
in general, I prefer moving these expressions out of a template, but that's a minor style preference for me.